README.md

```diff
@@ -80,6 +121,12 @@ can use:
     make stop
     ```
 
+
+> ``📝`` If you have created a custom environment file when starting the docker
+> containers you can export the path the env custom environment file by
+> using the `ENV_FILE` environment variable: `ENV_FILE=.env.dev make run`
```

Comment:
in terms of naming, i think `ENV_FILE` is pretty general. imo we could use a name like `FREVA_WEB_ENV` or something similar ...

Reply:
Ok, I used the workd `DOCKER_ENV_FILE`